01-hello-world/helloWorld.js

```diff
@@ -5,6 +5,7 @@
 function sayHello(name)
 {
     // Code goes here
+    console.log(`Hello, ${name}!`);
```

Comment:
It would be best to pull the template string out into its own variable, in case the format ever changed in the future.